### PR TITLE
Update promotion channel

### PR DESCRIPTION
**Pull request by epananth**
Make release/6.0  promote to .Net 6 Eng 

Issue -> https://github.com/dotnet/arcade/issues/7818

### CODE REVIEW

File: eng/validation/update-channel.ps1 [hunk 1]
```diff
@@ -2,7 +2,7 @@ Param(
   [string] $maestroEndpoint,
   [string] $barToken,
   [string] $apiVersion = "2018-07-16",
-  [string] $targetChannelName = ".NET Eng - Latest",
+  [string] $targetChannelName = ".NET 6 Eng",
```
Comment (MattGal) [inline eng/validation/update-channel.ps1:5]: Does it matter that darc get-channels shows a different casing (`".Net 6 Eng"`)?  Guessing not but figured I'd ask.

**Review by MattGal — APPROVED** [review]

File: eng/validation/update-channel.ps1 (hunk 1 above)
Comment (epananth) [reply eng/validation/update-channel.ps1:5]: I will update darc.. thanks for bringing this up :)

**Comment by riarenas** [comment]
Is this targetting the right branch? Won't this make main go to the wrong channel?

File: eng/validation/update-channel.ps1 (hunk 1 above)
Comment (MattGal) [reply eng/validation/update-channel.ps1:5]: Per Ricardo it doesn't actually matter and it's generally all caps, so I signed off.

**Comment by epananth** [comment]
closing this PR and recreatint

**Pull request closed without merging**